### PR TITLE
dark mode issue#272 solved 

### LINKED ISSUE #272: add logo & dark/light theme in signup page[Feat]:
### What feature?

i want to add gyanaguru logo in signup page onclick navigate to home page  & dark/light theme option - 
Screenshot (109)

We will see it in the login page just as it is -  

Screenshot (110)

_No response_

### Agreement to Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mandliyarajendra11**
## Description 📝

The sign-up page did not have a dark mode toggle component, and apart from that, if the website is in dark mode and you go to the sign-up page, some text was not visible due to certain classes.

## Changes 💬

I added the dark mode toggle component and made some changes in the CSS classes, adding some css class related to dark mode . Now, the dark mode option has been successfully added to it .

## Checklist ✅

- [ ] Reviewed the code changes thoroughly.
- [ ] Tested the changes locally and ensured they work as expected.
- [ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [ ] Code adheres to the project's style guidelines.
- [ ] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)

before when you on dark mode - 
Screenshot (114)

now  - 
Screenshot (115)

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

fixed  #272

### CODE REVIEW

**Review by lalitmee — CHANGES_REQUESTED** [review]
@mandliyarajendra11 
- add some screenshots of the changes in the pages
- remove the `package-lock.json` file from your file changes, as it is not required here.

**Comment by mandliyarajendra11** [comment]
@lalitmee check now

**Comment by mandliyarajendra11** [comment]
@PiyushKalyanpy  merge

File: database/firebase.tsx
Comment (PiyushKalyanpy) [inline database/firebase.tsx:1]: Do not include this file in PR.

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: You don't have to delete this file. 
Please make a new PR instead.
image

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (mandliyarajendra11) [reply package-lock.json:1]: okay

Comment (mandliyarajendra11) [reply package-lock.json:1]: > You don't have to delete this file. Please make a new PR instead. image

@PiyushKalyanpy 
check PR #306

**Pull request closed without merging**